src/routes/[...dir]/[zfile=generic]/+page.svelte

```diff
@@ -9,8 +9,20 @@
 		// Focus the iframe
 		iframe.contentWindow?.focus();
 	});
+
+	$: url = data.url.split('/');
+	$: title = getTitle();
+
+	function getTitle() {
+		return url[url.length - 1].split('?')[0];
+	}
```

Comment:
Così non sono sicuro che svelte si accorga dei cambiamenti.
Io farei piuttosto:
```suggestion

	$: title = getTitle(data.url);

	function getTitle(url: string) {
	    const parts = url.split("/");
		return parts[parts.length - 1].split('?')[0];
	}
```

Reply:
okay in effetti ha senso!